<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<div "featureFile">Feature File: src&#x2F;com&#x2F;test&#x2F;Test.feature</div>]]>
    </line>
    <line>
      <![CDATA[<div "feature">Feature: contact</div>]]>
    </line>
    <line>
      <![CDATA[<div "scenario">Scenario: Verify user is able to Add contact</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">Given  User is landed on App Home page (passed : 0s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">When  user taps on Add contact button (passed : 0s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">Then  user navigates to Add contact page (passed : 0s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">When  user Adds a contact using name, mobile and email (failed : 0s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">Then  verify the name under the list (skipped)</div>]]>
    </line>
  </reporter-output>
  <suite name="Suite1" duration-ms="21891" started-at="2016-12-28T12:22:43Z" finished-at="2016-12-28T12:23:05Z">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.run_cukes()[pri:0, instance:com.test.Runner@5025a98f]" name="run_cukes" class="cucumber.api.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test name="Test" duration-ms="21891" started-at="2016-12-28T12:22:43Z" finished-at="2016-12-28T12:23:05Z">
      <class name="com.test.Runner">
        <test-method status="FAIL" signature="run_cukes()[pri:0, instance:com.test.Runner@5025a98f]" name="run_cukes" duration-ms="21875" started-at="2016-12-28T17:52:43Z" description="Runs Cucumber Features" finished-at="2016-12-28T17:53:05Z">
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: java.lang.NullPointerException
	at cucumber.api.testng.TestNGCucumberRunner.runCukes(TestNGCucumberRunner.java:48)
	at cucumber.api.testng.AbstractTestNGCucumberTests.run_cukes(AbstractTestNGCucumberTests.java:14)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:189)
	at cucumber.api.testng.AbstractTestNGCucumberTests.run(AbstractTestNGCucumberTests.java:19)
	at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:201)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:642)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
Caused by: java.lang.NullPointerException
	at com.test.StepDef.AddDetails(StepDef.java:58)
	at âœ½.When user Adds a contact using name, mobile and email(src/com/test/Test.feature:7)
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
            <line>
              <![CDATA[<div "featureFile">Feature File: src&#x2F;com&#x2F;test&#x2F;Test.feature</div>]]>
            </line>
            <line>
              <![CDATA[<div "feature">Feature: contact</div>]]>
            </line>
            <line>
              <![CDATA[<div "scenario">Scenario: Verify user is able to Add contact</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">Given  User is landed on App Home page (passed : 0s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">When  user taps on Add contact button (passed : 0s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">Then  user navigates to Add contact page (passed : 0s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">When  user Adds a contact using name, mobile and email (failed : 0s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">Then  verify the name under the list (skipped)</div>]]>
            </line>
          </reporter-output>
        </test-method> <!-- run_cukes -->
      </class> <!-- com.test.Runner -->
    </test> <!-- Test -->
  </suite> <!-- Suite1 -->
</testng-results>
